src/main/java/com/zendesk/maxwell/row/RowIdentity.java

```diff
@@ -0,0 +1,138 @@
+package com.zendesk.maxwell.row;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.*;
+
+public class RowIdentity implements Serializable {
+	private final List<AbstractMap.SimpleImmutableEntry<String, Object>> primaryKeyColumns;
+
+	private String database;
+	private String table;
+	private JsonSerializer serializer = new JsonSerializer();
+
+	// slightly less verbose pair construction
+	public static AbstractMap.SimpleImmutableEntry<String, Object> pair(String key, Object value) {
+		return new AbstractMap.SimpleImmutableEntry<>(key, value);
```

Comment:
If we need immutable key value pairs, `org.apache.commons.lang3.tuple` package has `ImmutablePair`, which may help reduce verbosity.

```java
ImmutablePair<Integer, String> pair = new ImmutablePair<>(2, "Two");
Integer key = pair.getKey();
String value = pair.getValue();
```

Reply:
There's only one instance of the awkward `AbstractMap.SimpleImmutableEntry` now that I'm using the interface in most places, so this is less of an issue.